src/cdk/overlay/overlay.md

```diff
@@ -1,3 +1,88 @@
 ### Overlay
 
-A service used to manage overlays.
\ No newline at end of file
+The `Overlay` service provides a way to open floating panels on the screen.
+
+### Creating overlays
+Calling `overlay.create()` will return an `OverlayRef` instance. This instance is a handle for
+managing that specific overlay.
+
+The `OverlayRef` _is_ a `PortalHost`- once creates, content can be added by attaching a `Portal`.
+See the documentation on portals for further information.
```

Comment:
Link to the portal docs (if we have any)?

Reply:
They don't exist yet; I'm planning to do a pass and ass cross-references once I have more of the content authored